packages/cli/package.json

```diff
@@ -26,8 +26,8 @@
     "@rushstack/ts-command-line": "^4.3.13",
     "ansi-colors": "^4.1.1",
     "git-rev-sync": "^2.0.0",
-    "martinez-polygon-clipping": "^0.7.0",
     "p-limit": "^2.2.1",
+    "polygon-clipping": "^0.14.3",
```

Comment:
Whats the difference?

Reply:
The other broke when zoomed in a lot. This one seems more correct is based off the other one but uses later dependencies but was released longer ago